IntrospectTests/AppKitTests.swift

```diff
@@ -4,6 +4,8 @@ import XCTest
 import SwiftUI
 @testable import Introspect
 
+let TIMEOUT: TimeInterval = 2 // just to be really safe on CI
```

Comment:
Can we use swift style guide syntax for this? Shouldn't be all uppercase and should probably be in an enum `Constants` inside `TestUtils` as a static let.